delivery/routes.py

```diff
@@ -13,21 +13,19 @@ def get(self):
 
     def post(self):
         content = request.json
-        self.restaurants.create(self.schema.load(content))
-        return jsonify([self.schema.dump(restaurant) for restaurant in self.restaurants.get_all()]), 201
+        return jsonify(self.schema.dump(self.restaurants.create(self.schema.load(content)))), 201
```

Comment:
it might be more readable if split into lines:
```python
restaurant = self.schema.load(request.json)
created_restaurant = self.repository.create(restaurant)
return jsonify(self.schema.dump(created_restaurant)), 201
```

Reply:
fixed